src/components/Spells/SpellBySchoolAndLevel.vue

```diff
@@ -0,0 +1,68 @@
+<template>
```

Comment:
Is this structure the best way? I might actually collapse these spells views into one that conditionally adds parameters to the apollo query